packages/client/hmi-client/src/components/widgets/tera-asset-block.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<Panel :toggleable="isToggleable">
+	<Panel :toggleable="isToggleable" :class="useDefaultStyle ? 'asset-panel' : ''">
```

Comment:
This is the vue way to assign class based on a boolean:
`:class="{ 'asset-panel': useDefaultStyle }"`